### PR TITLE
Deeper comparator

### PR DESCRIPTION
**Pull request by AWare**
Improve the comparator by checking at a field level.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,11 +36,12 @@
     "run:querier": "lambda-local -l dist/querier.js -e test/querier.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
     "run:fetcher": "lambda-local -l dist/fetcher.js -e test/fetcher.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
     "run:sf": "lambda-local -l dist/salesforce_downloader.js -e test/fetcher.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
-    "run:comparator": "lambda-local -l dist/comparator.js -e test/fetcher.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
+    "run:comparator": "lambda-local -l dist/comparator.js -e test/fetcher.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"PROD\\\"}",
```
Comment (pvighi) [inline package.json]: why PROD?

File: flow-typed/npm/babel-cli_vx.x.x.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```
Comment (pvighi) [inline flow-typed/npm/babel-cli_vx.x.x.js]: maybe all of these flow changes should have been in a separate PR. Just a comment for next time, I'm not saying you should change it

File: package.json (hunk 1 above)
Comment (AWare) [inline package.json]: Sorry, shouldn't have committed.

File: flow-typed/npm/babel-cli_vx.x.x.js (hunk 2 above)
Comment (AWare) [inline flow-typed/npm/babel-cli_vx.x.x.js]: This was entirely auto-generated and only really affects linting so I didn't think it was worth a PR on it's own.

Comment (pvighi) [inline flow-typed/npm/babel-cli_vx.x.x.js]: ok but if they are auto generated why do we need them in the repo? is something external needed to generate them or something?

Comment (AWare) [inline flow-typed/npm/babel-cli_vx.x.x.js]: Yeah, they let flow type check code which uses those libraries. I should have merged it earlier, but I wasn't using flow properly.

**Pull request merged**